### DISCUSSION TITLE
Adding Extra Parameters to Matrix.to links

### DISCUSSION THREAD

**Discussion by daniellekirkwood** [element-hq/element-meta#935] [Product Feedback]
### Your use case

As part of the the Threads project, we're discussing having extra parameters in the Matrix.to link so that client will know (and open) a Thread Panel, Thread List, or Timeline view (and then have the context for which they were sent a link). 

This would be an improvement for Threads but needs discussion at a much higher level...

**Comment by daniellekirkwood** [comment]
@janogarcia feel free to add extra info to this issue! 
@gsouquet I think we discussed in standup today that you would raise this for discussion in the appropriate room.

**Comment by germain-gg** [comment]
This has been discussed in a public matrix room, https://matrix.to/#/!ekJXCOhsmAWQUMhDgn:riot.ovh/$1639752472393594TxiJX:matrix.org

It looks like people would be open to it and the path forward would be to open an MSC to extend https://github.com/matrix-org/matrix-doc/pull/2312 which defines the Matrix URI scheme

**Comment by daniellekirkwood** [comment]
Internal document for future reference: https://docs.google.com/document/d/1EKoRWfVxTTkDgAZxzTPh2x_BrCASIm43cwJWbixSL24/edit

We put the decision on pause and deprioritised this. I'll move it to Discussions for future reference.
